prisma/schema.prisma

```diff
@@ -0,0 +1,23 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:./dev.db"
+}
+
+model Duels {
+  userName      String   @unique
```

Comment:
Using `user.username` has the potential to not be unique. We should be using the user's snowflake (`user.id`) instead. This will require updates to the duel, duelStats and updateUserScore functions in `duel.ts`